plan/entities.go

```diff
@@ -69,6 +98,22 @@ func (b Behaviors) Len() int           { return len(b) }
 func (b Behaviors) Swap(i, j int)      { b[i], b[j] = b[j], b[i] }
 func (b Behaviors) Less(i, j int) bool { return b[i].Name < b[j].Name }
 
+func (b Behaviors) validateAndApplyFilters(filtersByBehavior map[string][]Filter) error {
```

Comment:
call this `attachFilters` because that's really what it's doing. It's attaching the filters to the behaviors.

The filters are 'applied' later when we decide whether to skip a test case.